.github/workflows/build.yml

```diff
@@ -42,7 +44,10 @@ jobs:
           export CGO_ENABLED=1
           export CC=${{ matrix.cc }}
           go build -ldflags "-X github.com/massalabs/thyra/pkg/config.Version=${{env.VERSION}}" -o massastation_${{ matrix.target }}_${{ matrix.arch }} ./cmd/massastation/
-      - name: Upload Package
+          if [ ${{ matrix.target }} != "windows" ]; then
+            fyne package -name MassaStation -icon ./cmd/massastation/logo.png -appID net.massalabs.massastation -exe massastation_${{ matrix.target }}_${{ matrix.arch }}
+          fi
```

Comment:
maybe you can put this in a different step of the workflow ? with a proper name

Reply:
The issue is that, if we move this to another workflow, we need to reinstall all the dependencies, which takes like 5min per OS, and we end up with a CI that needs 15min to run (we are already at 10min for the `api` workflow that does the build and packaging).
We could reduce a bit the time the CI takes to pass by removing the installer building, but we are planning to try to test the installers in the CI.

Reply:
i just said differnt step, but same workflow

Reply:
Oh sorry! 
I kept those in the same step to avoid having to redo the exports.
I will check if it's possible to do it in a clean way